plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/FirUtils.kt

```diff
@@ -30,4 +31,9 @@ val KtSourceElement?.asPosition: Position
     get() = when (this) {
         null -> Position.NoPosition
         else -> Position.Wrapped(this)
+    }
+val FirBasedSymbol<*>?.asInfo: Info
```

Comment:
Would wrapping this in a `SourceRole` for consistency not make more sense?

Reply:
Yeah, I was unsure about that tbh. I'm going to do that then.